core/src/avm2.rs

```diff
@@ -154,17 +153,13 @@ impl<'gc> Avm2<'gc> {
     /// Dispatch an event on an object.
     ///
     /// The `bool` parameter reads true if the event was cancelled.
-    pub fn dispatch_event(
+    pub fn dispatch_event_object(
```

Comment:
Bikeshedding: seems like renaming the fn is a little wordy to me, I prefer the shorter name, and it'd be obvious upon seeing the signature that it takes an Object.

Reply:
TBH: I renamed it early to make "function not found" compiler errors into a simple TODO list of call sites to update :)
I can change it back now, if you think this naming is too explicit.